src/ScriptEditor/NetscriptDefinitions.d.ts

```diff
@@ -4776,77 +4775,60 @@ export type NS = {
   hackAnalyzeChance(host: string): number;
 
   /**
-   * Estimate the number of grow threads needed to grow a server by a certain multiplier.
+   * Calculate the number of threads needed to multiply funds on a target host by a given multiplier.
```

Comment:
Please leave this as "Estimate." I feel pretty strongly about this. "Calculate" implies that this gives the correct, exact answer, and it does not.

Reply:
It is calculating. It's calculating an exact number of threads so that the multiplicative portion of growth is exactly what the player requested.

It's not calculating the overall effective multiplier because there is an additive step and a multiplicative step to server growth, and this is just calculating an exact threadcount for the actual multiplying portion.

I redid some more of the doc, including ns.grow, to make it even more clear that growthAnalyze is calculating only for the multiplier, and that there is also an additive portion of growth that goes into the overall effective multiplier.